### ISSUE TITLE
Sometimes getting a crash while exploring

### ISSUE THREAD

**Issue by belathus** [mcenderdragon/Futurepack-API#62]
https://cdn.discordapp.com/attachments/176845387431149568/242872779513397248/crash-2016-11-01_00.55.19-server.txt

**Comment by mcenderdragon** [comment]
This is some difficult one, because it is not only me who is involved. Also Primal (PrimalCore-0.1.10.jar). This happened because both of us a generating stuff and mine has replaced his, and thats crashed the game.

**Comment by mcenderdragon** [comment]
OK nope its not Primal. You got somehow broken loot tables. something is spawning vanilla chest who try to use loottables that ares omehow not working. (It uses a function for an Item Entrys that does not exists).
This bug was reaveled through my mod because I replaced a chest while spawning a large structure.

**Comment by belathus** [comment]
Huh. When I reported this, it was happening rather frequently. We updated a
few mods since then and I don't think I've seen the crash since. Maybe we
can consider this resolved.

On Thu, Nov 3, 2016, 1:15 PM mcenderdragon <notifications@github.com> wrote:

OK nope its not Primal. You got somehow broken loot tables. something is
spawning vanilla chest who try to use loottables that ares omehow not
working. (It uses a function for an Item Entrys that does not exists).
This bug was reaveled through my mod because I replaced a chest while
spawning a large structure.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/mcenderdragon/Futurepack-API/issues/62#issuecomment-258210282>,
or mute the thread
